assign-doi-datacite/src/main/java/no/unit/nva/doi/datacite/clients/exception/SetLandingPageException.java

```diff
@@ -1,6 +1,6 @@
 package no.unit.nva.doi.datacite.clients.exception;
 
-import no.unit.nva.doi.datacite.clients.models.Doi;
+import no.unit.nva.doi.models.Doi;
```

Comment:
Uses `publication-doi-type` module now. Currently with publication service due to it also is convenient to be together with `doi-commons`.